### PR TITLE
Draft Agenda: Collaborator Summit Montreal 2019

### PR DESCRIPTION
**Pull request by WaleedAshraf**
Draft Agenda.

@mcollina @keywordnew @jorydotcom @ladyleet

### CODE REVIEW

File: 2019-12-Montreal/agenda.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+## Organizers
+
+- Tracy Lee ([@ladyleet](https://github.com/ladyleet))
+- Jory ([@jorydotcom](https://github.com/jorydotcom))
+- TBA
+
+## Attendance
+
+Anyone can come, but we will not be explicitly onboarding at Collab Summit Berlin. Conversations can move fast as working groups have a lot of context.
```
Comment (cjihrig) [inline 2019-12-Montreal/agenda.md]:
```suggestion
Anyone can come, but we will not be explicitly onboarding at Collab Summit Montreal. Conversations can move fast as working groups have a lot of context.
```

File: 2019-12-Montreal/agenda.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+TBA
+
+## Collaborator Summit Day One: 13th December, 2019
+
+### Agenda
+TBA
+_To speak/present a topic/do a sesssion, please open a issue in repo. Check https://github.com/openjs-foundation/summit/issues/184 for more details.
+
+
+## Collaborator Summit Day Two: 14st December, 2019
```
Comment (cjihrig) [inline 2019-12-Montreal/agenda.md]:
```suggestion
## Collaborator Summit Day Two: 14th December, 2019
```

File: 2019-12-Montreal/agenda.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# Collaborator Summit Montreal 2019 - December 13th and 14th 🇨🇦
+
+This is a live document that will be constantly updated to match the proposed agenda.
+
+A collaborator summit is a mix of presentations and interactive sessions lead by the Teams, Working Groups, and Projects that participate in the summit. For interactive sessions, our time is best served if a the team/working groups/project participate in good numbers (quorum etc).
+
+__We can change everything__, so if you think a better structure will suit us better, let us know in #201, or just send a PR!
```
Comment (trivikr) [inline 2019-12-Montreal/agenda.md]:
```suggestion
__We can change everything__, so if you think a better structure will suit us better, let us know in [#202](https://github.com/openjs-foundation/summit/issues/202), or just send a PR!
```

File: 2019-12-Montreal/agenda.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  09:00 - 18:00
+
+### Room Setup - both days
+TBA
+
+## Collaborator Summit Day One: 13th December, 2019
+
+### Agenda
+TBA
+_To speak/present a topic/do a sesssion, please open a issue in repo. Check https://github.com/openjs-foundation/summit/issues/184 for more details.
```
Comment (trivikr) [inline 2019-12-Montreal/agenda.md]:
```suggestion
_To speak/present a topic/do a sesssion, please open a issue in repo. Check https://github.com/openjs-foundation/summit/issues/184 for more details._
```

File: 2019-12-Montreal/agenda.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 1.4, available [HERE](https://www.contributor-covenant.org/version/1/4/)
+
+### Appendix A: The report@nodejs.org Alias
+
+Note: At the current time, all email sent to the [report@nodejs.org](mailto:report@nodejs.org) email alias is copied to all members of the [Node.js Moderation Team](https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#current-members).
+
+
+------------------------
+
+✨ Super stoked to see everyone in May! ✨
```
Comment (ruyadorno) [inline 2019-12-Montreal/agenda.md]:
```suggestion
✨ Super stoked to see everyone in December! ✨
```

**Comment by WaleedAshraf** [comment]
I'll merge this for now, and open a new one for adding agenda.

**Pull request merged**